cache/action.yml

```diff
@@ -8,9 +8,24 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Bootstrap devx shell
+      uses: input-output-hk/actions/devx@latest
+      with:
+        platform: 'x86_64-linux'
+        target-platform: ''
+        compiler-nix-name: ${{ inputs.ghc_version }}
```

Comment:
I should have called `ghc_version` rather `compiler-nix-name` ... if I rename it I should also change it here https://github.com/IntersectMBO/cardano-base/blob/master/.github/workflows/haskell.yml#L158

Reply:
(or maybe the action should takes as input also `platform`, `target-platform`, `minimal` and `iog`? even if all GitHub Codespace needs HLS, runs on `x86_64-linux` and likely want to offers `-iog` flavor)

Reply:
for now let's try to keep it simple and build a reliable PoC, then expand.

We won't do target platforms yet, flavours _could_ be an issue, but as long as we keep the one in CI and the one in the devcontainer the same it should be ok.